readme.md

```diff
@@ -296,6 +296,10 @@ choosing. If the second argument is omitted, the application will use a built-in
 - [OpenSSL](https://www.openssl.org/) or [BoringSSL](https://boringssl.googlesource.com/boringssl/) (optional; required
   for TLS support (strongly recommended))
 
+   ```bash
+   sudo apt install libgoogle-glog-dev libgflags-dev libboost-all-dev libssl-dev 
```

Comment:
Since this is cross platform, maybe some text that says "In Linux, you can easily install the above dependencies with the following command:"

Reply:
(I also believe these requirements are only needed in the cmake flow. Bazel will pull them in automatically)